docs/rootless.md

```diff
@@ -26,7 +26,12 @@ See also the [example manifests](#Kubernetes).
 
 ### Bottlerocket OS
 
-Needs to run `sysctl -w user.max_user_namespaces=N` (N=positive integer, like 63359) on the host nodes.
+Needs to set the max user namespaces to a positive integer, through the [API settings](https://github.com/bottlerocket-os/bottlerocket#kernel-settings):
+
+```toml
+[settings.kernel.sysctl]
+"user.max_user_namespaces" = "16384"
+```
```

Comment:
Does that work for EKS?

Reply:
Yes, the Bottlerocket API is independent of the orchestrator, so this configuration will work in EKS and ECS.

Reply:
Couldn't find out how to set this configuration with eksctl

Reply:
The eksctl documentation doesn't expand too much on how to configure Bottlerocket's API, but they do mention the `bottlerocket` field for Bottlerocket nodes:

https://eksctl.io/usage/custom-ami-support/?h=bottlerocket#bottlerocket-custom-ami-support

So, the configurations will look like this in the yaml file:

```yaml
  nodeGroups:
  - name: bottlerocket-ng
    ami: ami-custom1234
    amiFamily: Bottlerocket
    bottlerocket:
      enableAdminContainer: true
      settings:
        kernel:
          sysctl:
            "user.max_user_namespaces": 16384
```

Do you think an example like this will help in this documentation?

Reply:
Yes, but I'd expect that the yaml doesn't require `ami-custom1234`.

Reply:
An alternative would be to call `apiclient` in a daemonset

https://github.com/bottlerocket-os/bottlerocket/blob/v1.13.1/sources/api/apiclient/README.md#json-input

Reply:
I added a configuration file to create a node group with the expected `sysctl` knobs values.